tests/frontend/github_repo_info_controller_test.js

```diff
@@ -15,18 +15,23 @@
 
 import { Application } from "@hotwired/stimulus";
 import GitHubRepoInfoController from "../../warehouse/static/js/warehouse/controllers/github_repo_info_controller";
+import GitHubRepoStatsController from "../../warehouse/static/js/warehouse/controllers/github_repo_stats_controller";
```

Comment:
I'm trying to decide if it makes sense to split this file into two separate test modules (this existing file for `github_repo_info_controller`, and potentially a separate file for `github_repo_stats_controller`).

Their behaviour is fairly tightly-coupled though - I think by necessity, since they need to agree on the format of the `stats` object used during communication.

Reply:
Thanks for calling this out - it's making me wonder if this is worth it.

We're adding another controller to handle the existence of two controllers in the HTML and synchronize their output via the new Outlets API. That is indeed clever, and I've confirmed that it works.

I feel that the "right" solution would be to reduce the double-inclusion of the controller in the HTML, this could be considered a `TODO` to resolve/remove at a later date.

BUT if we're to proceed with this for now, the test should be updated to reflect the behavior we're changing - notably that there are two `github-repo-info` elements, and assert in the successful test that we only make a single fetch call (i.e. leave `expect(fetch.mock.calls.length).toEqual(3);` should not change as a result of the second HTML fragment being included.